plugins/woocommerce-blocks/assets/js/data/cart/thunks.ts

```diff
@@ -143,13 +146,19 @@ export const applyExtensionCartUpdate =
 	};
 
 /**
- * Syncs the cart with the iAPI store.
+ * Fetch the cart but avoid triggering the event that syncs with the
+ * Interactivity API store to avoid infinite loops.
  *
+ * @param {QuantityChanges} quantityChanges The quantity changes data included in the sync event.
  * @throws Will throw an error if there is an API problem.
  */
-export const syncCartWithIAPIStore =
-	() =>
-	async ( { dispatch }: CartThunkArgs ) => {
+export const syncCartwithIAPIStore =
```

Comment:
nitpick on casing here
```suggestion
export const syncCartWithIAPIStore =
```